rmw_connext_cpp/src/functions.cpp

```diff
@@ -361,6 +410,14 @@ rmw_create_publisher(
     dds_publisher->get_instance_handle(), topic_name, type_name, EntityType::Publisher);
   node_info->publisher_listener->trigger_graph_guard_condition();
 
+#ifdef DISCOVERY_DEBUG_LOGGING
+  fprintf(stderr, "******* Creating Publisher Details: ********\n");
+  fprintf(stderr, "Publisher partition %s\n", publisher_qos.partition.name[0]);
+  fprintf(stderr, "Publisher topic %s\n", topic_writer->get_topic()->get_name());
+  fprintf(stderr, "Publisher address %p\n", static_cast<void *>(dds_publisher));
+  fprintf(stderr, "******\n");
+#endif
```

Comment:
I'd say there is no need, because when the logging macros are available we'll have to audit all of the code for uses of printf and fprintf to see if they should be replaced.